docs/lms-service-api-document.md

```
@@ -2412,6 +2517,97 @@ organisationid: <organisation-id>
 ```
 
+#### Clone Lesson
+
+**Endpoint**: `POST /lessons/{lessonId}/clone`
+
+**HTTP Method**: POST
+
+**Description**: Clone a lesson with all its media and associated files
+
+**Headers**:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)